randovania/games/am2r/pickup_database/pickup-database.json

```diff
@@ -401,7 +401,7 @@
             "offworld_models": {
                 "prime1": "Missile",
                 "prime2": "MissileLauncher",
-                "dread": "powerup_icemissile"
+                "dread": "powerup_missile"
```

Comment:
didnt you also already have a super launcher? or do you wanna put that off to another PR?

Reply:
does it not already use the vanilla super model?

Reply:
im dumb, meant super expansion

Reply:
nope. I still need to get the green M10 tank created. and merged into odr. probably gonna be another  PR unless dunc wants me to bundle everything into one.